src/index.js.flow

```diff
@@ -0,0 +1,17 @@
+/* flow strict */
+
+declare module 'auto-check-element' {
+  declare export default class AutoCheckElement extends HTMLElement {
+    boundCheck: () => void;
+    input: HTMLInputElement;
```

Comment:
These two entries are not supported public API so they can be removed from the type refs here. If we want to support accessing the managed input element, `input` could be defined as a getter.

Reply:
Yup! That's what I essentially did in this PR but then split that out into #14. I really wanted to represent the code in the repo and currently `input`, `check` and `boundCheck` are accessible via the element.

Omitting properties that are accessible from the flow definition seems a bit off to me but I don't mind removing these in this PR since I do move those properties out the element in #14.